src/services/factories/tokenList.ts

```diff
@@ -15,7 +15,7 @@ export function getTokensFactory(
   factoryParams: {
     tokenListApi: TokenList
     exchangeApi: ExchangeApi
-    tcrApi: TcrApi
+    tcrApi: TcrApi | undefined
```

Comment:
Same as :point_down: ?

```suggestion
    tcrApi?: TcrApi
```